### PR TITLE
Skip python script from global hub sonarcloud

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main-presubmits.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-stolostron-multicluster-global-hub-main-sonarcloud
     rerun_command: /test sonarcloud
-    skip_if_only_changed: ^doc/|^samples/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
+    skip_if_only_changed: ^doc/|^samples/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.py$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
     spec:
```
Comment (morvencao) [inline ci-operator/jobs/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main-presubmits.yaml:68]: we don't need `.md` any more?

Comment (yanmxa) [reply ci-operator/jobs/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main-presubmits.yaml:68]: The `.md` is still here. We wrote it repeatedly.

**Review by morvencao — COMMENTED** [review]
/lgtm

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**